lua/ii/disting.lua

```diff
@@ -148,6 +198,46 @@ do return
     , retval = { 'state', u8 }
     , docs = '0-based loop index'
     }
+  , { name = 'dual_parameter'
```

Comment:
wonder if we can abbreviate `parameter` to `param`? this obviously applies to all the above elements too.

similarly, could `algorithm` be `algo`?